sdk/eventhub/ci.yml

```diff
@@ -10,4 +10,4 @@ variables:
   ProjectFile: sdk/eventhub/Microsoft.Azure.EventHubs.sln
 
 jobs:
-- template: ../../.azure-pipelines/templates/jobs/archetype-sdk-client.yml
\ No newline at end of file
+- template: templates/jobs/archetype-sdk-client.yml
```

Comment:
How does it resolve this relative path?

Reply:
Whoops - fixed this. Makes me wonder whether I should introduce a global which is $(Eng.Path) etc.